### PR TITLE
NW6| Rabia Avci | React Module Project | Feature Card - Week1 

### PR DESCRIPTION
**Pull request by RbAvci**
implemented Card
add Css styling for card
implemented test for each props of Card components

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

The Card component is created to hold different kinds of information together, such as picture, title, and links. It's a reusable small part of our cyf hotel project.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: src/components/Card/Card.test.jsx [hunk 1]
```diff
@@ -2,8 +2,24 @@ import Card from "./Card.jsx";
...
 
+
 describe("Card", () => {
-  it("should have a test, please write one", () => {
-    return true;
+  it("renders Card component with provided props", () => {
+    const testData = {
+      title: "Test Title",
+      url: "https://example.com",
+      image: "https://example.com/image.jpg",
```
Comment (fikretellek) [inline src/components/Card/Card.test.jsx:11]: well done rabia  its working perfect. just note that img is not loading

**Review by fikretellek — APPROVED** [review]
Well done Rabia. Your code looks well written and perfect.

File: src/components/Card/Card.scss [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+//   border-color: var(--color--accent);
+
+//   &__title {
+//     grid-area: title;
+//   }
+//   &__image {
+//     grid-area: image;
+//     mix-blend-mode: overlay;
+//   }
+// }
```
Comment (Ara225) [inline src/components/Card/Card.scss:16]: I'd strip this commented out code out https://syllabus.codeyourfuture.io/guides/code-style-guide

File: src/components/Card/Card.scss [hunk 3]
```diff
@@ -1,16 +1,53 @@
...
-    ". title ." auto / var(--space--gap) 1fr var(--space--gap);
-  border: var(--border);
-  border-color: var(--color--accent);
+  width: 300px;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  background-color: #fff;
+}
```
Comment (Ara225) [inline src/components/Card/Card.scss:27]: This card looks so professional, with the subtle styles and nicely rounded corners :)

File: src/components/Card/Card.scss [hunk 4]
```diff
@@ -1,16 +1,53 @@
...
+  padding: 8px 16px;
+  background-color: #007bff;
+  color: #fff;
+  text-decoration: none;
+  border-radius: 4px;
+  transition: background-color 0.3s ease;
+}
+
+.card a:hover {
+  background-color: #0056b3;
```
Comment (Ara225) [inline src/components/Card/Card.scss:52]: Adding a hover style is really important for anything you style a link in a non standard way, good to see you've done it.

However the background color this sets is way too close to the text colour, so the text is diffcult to read. The text needs to stay clear and readable no matter what so the user can interact. This link has some relvant information which might be of intrest https://css-tricks.com/understanding-web-accessibility-color-contrast-guidelines-and-ratios/

**Pull request merged**